clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/security/AmazonClientProvider.java

```diff
@@ -279,7 +279,7 @@ public AmazonEC2 getAmazonEC2(String accountName, AWSCredentialsProvider awsCred
     return awsSdkClientSupplier.getClient(AmazonEC2ClientBuilder.class, AmazonEC2.class, accountName, awsCredentialsProvider, region);
   }
 
-  public AmazonECS getAmazonEcs(String accountName, AWSCredentialsProvider awsCredentialsProvider, String region) {
+  public AmazonECS getAmazonECS(String accountName, AWSCredentialsProvider awsCredentialsProvider, String region) {
```

Comment:
I'd rather not change classes in the amazon module without compelling reason, as we have to go through the pain of making a pull request

Reply:
Restored